vscode/microsoft-kiota/src/utilities/deep-linking.ts

```diff
@@ -28,6 +29,7 @@ export function transformToGenerationConfig(deepLinkParams: Partial<IntegrationP
         break;
       case "apimanifest":
```

Comment:
This still doesn't work for case "openAI". You can test this with 
[test.md](https://github.com/user-attachments/files/17041190/test.md)